tools/setupResourceScripts/setup_tdr_resources.py

```diff
@@ -227,14 +227,9 @@ def delete_dataset_if_exists(name, clients):
 
 
 def add_snapshot_builder_settings(clients, dataset_id, directory, snapshot_builder_settings_file):
-  with open(
-    os.path.join("files", directory, snapshot_builder_settings_file)) as dataset_schema_json:
-    # In order to use the client API, we need to publish the updated python client library
-    response = requests.post(f"{clients.api_client.configuration.host}/api/repository/v1/datasets/{dataset_id}/snapshotBuilder/settings", json=json.load(dataset_schema_json), headers={"Authorization": f"Bearer {clients.api_client.configuration.access_token}"})
-    if response.status_code != 200:
```

Comment:
I assume that error checking and logging is now handled by the generated client code?

Reply:
Yes, it should. I'm not sure it's a particularly pretty error message but it surfaces the error message.